agent/engine/service_connect/manager_linux.go

```diff
@@ -32,6 +41,8 @@ const (
 	defaultRelayPathHost       = "/var/run/ecs/service_connect/relay/"
 	defaultRelayFileName       = "envoy_xds.sock"
 	defaultEndpointENV         = "APPMESH_XDS_ENDPOINT"
+	defaultEndpointPortENV     = "APPNET_MANAGEMENT_PORT"
+	defaultEndpointPort        = "443"
```

Comment:
These don't make any sense now that we changed to APPMESH_XDS_ENDPOINT I believe

Reply:
True, the endpoint we get from `DiscoverPollEndpoint` will already have `https://` and implies port 443 by default.